### DISCUSSION TITLE
About sound and light

### DISCUSSION THREAD

**Discussion by taara67** [iRobotEducation/create3_docs#447] [Questions]
### How are you connecting to your Create 3?

Wi-Fi (ROS 2)

### Which firmware version is installed on your robot?

G4.4

### Which RMW is your robot running?

I don't know

### Describe your question.

Hello
I started using Create3.
The noise is so loud that it becomes a nuisance in my laboratory. Is there a way to reduce or adjust the default sounds and lights, such as the startup sound?

**Comment by shamlian** [comment]
I'm sorry that you find the Create robot's audio and visual feedback to be disruptive. These indicators provide important and accessible feedback about the state of the robot (charging, processor loading, network status, payload status, etc.). There is no way to disable them in software.

I suppose, speaking as an individual with small children who have toys I'm not always fond of and _not_ under my capacity as an iRobot employee, if you were suitably motivated, you could (carefully!) unplug the robot's internal speaker and tape over the LED indicators, though I cannot guarantee the former will be easy or that the latter will not cause other disruption.

So that we can consider your team's needs in our future product decisions, can you tell me more about the lab you're in, and the ways in which the robot's sound and lights cause a problem?

**Reply by slowrunner** [reply]
> So that we can consider your team's needs in our future product decisions, can you tell me more

Not OP but have same need for ability to turn off sounds.  I have two 24/7 robots, each honors “quiet hours” logging all silenced TTS and alerts without the accompanying sound.  

I plan to create a 24/7 home robot on the Create3.  The physical noise / clunking when undocking is pretty disturbing but I need a way to be sure no beeps or alert sounds are played while the family is sleeping.
